downstream/assemblies/hub/assembly-managing-cert-valid-content.adoc

```diff
@@ -43,19 +51,28 @@ For instructions on certifying your collection, see the Ansible certification po
 [discrete]
 == How does the joint support agreement on Certified Collections work?
 
-If a customer raises an issue with the Red Hat support team about a certified collection, Red Hat support assesses the issue and checks whether the problem exists within Ansible or Ansible usage.
+If a customer raises an issue with the Red Hat support team about a certified collection, Red Hat support assesses the issue and checks whether the problem is with Ansible or Ansible usage.
 They also check whether the issue is with a certified collection.
-If there is a problem with the certified collection, support teams transfer the issue to the vendor owner of the certified collection through an agreed upon tool such as TSANet.
+If there is a problem with the certified collection, support teams transfer the issue to the vendor owner of the certified collection through an agreed-upon tool such as TSANet.
 
 [discrete]
 == Can I create and certify a collection containing only Ansible Roles?
 
 You can create and certify collections that contain only roles.
-Current testing requirements are focused on collections containing modules, and additional resources are currently in progress for testing collections only containing roles.
+Current testing requirements are focused on collections containing modules, and additional resources are currently in progress for testing collections containing only roles.
 Contact ansiblepartners@redhat.com for more information.
+////
+
+You can use {HubNameMain} to distribute the relevant {CertifiedColl}s to your users by creating a requirements file or a synclist. We recommend using a requirements file to install collections to your {HubName}, as synclists can only be managed by users with platform administrator privileges. 
```

Comment:
Remove "We recommend" and replace with "Use a requirements file to ..."

Reply:
updated